casa/IO/MultiFileBase.cc

```diff
@@ -93,23 +158,45 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     return nf;
   }
 
-  void MultiFileBase::flush()
+  Bool MultiFileBase::flush()
   {
     // Flush all buffers if needed.
-    for (vector<MultiFileInfo>::iterator iter=itsInfo.begin();
-         iter!=itsInfo.end(); ++iter) {
-      if (iter->dirty) {
-        writeDirty (*iter);
+    for (auto info : itsInfo) {
```

Comment:
This creates copies of the info object, why is this change in semantic needed?

Reply:
Changed to &